lib/playground.dart

```diff
@@ -579,6 +582,22 @@ ${info['libraryName'] == null ? "" : "**Library:** ${apiLink == null ? info['lib
         messageSpan.classes.add('message');
         messageSpan.text = issue.message;
         e.children.add(messageSpan);
+        source.offset = issue.charStart;
+        dartServices.fix(source).timeout(serviceCallTimeout).then((FixesResponse fixResponse) {
+          if (fixResponse.fixes.isNotEmpty) {
```

Comment:
Ah, ok. I understand now.

I don't believe we should be making service calls while we're building up the UI for analysis issues. Let's comment out or remove this code for now, and figure out if there's a solution that'll provide @lukechurch with the UI he wants.

It would be nice if the service call that sends us analysis issues, could provide a boolean on each issue telling us whether it had any quick fixes. This might be O(1) for the services layer to provide. I think a quick fix is always associated with a type of error.